prog/weaver/main.go

```diff
@@ -497,6 +517,31 @@ func (nopPacketLogging) LogPacket(string, weave.PacketKey) {
 func (nopPacketLogging) LogForwardPacket(string, weave.ForwardPacketKey) {
 }
 
+func configureProxy(version string, defaultDockerHost string) *weaveproxy.Config {
+	proxyConfig := weaveproxy.Config{
+		Version:      version,
+		Image:        getenvOrDefault("EXEC_IMAGE", "weaveworks/weaveexec"),
+		DockerBridge: getenvOrDefault("DOCKER_BRIDGE", "docker0"),
+		DockerHost:   defaultDockerHost,
+	}
+	mflag.BoolVar(&proxyConfig.Disabled, []string{"-no-proxy"}, false, "instruct Weave not to start the Docker proxy")
```

Comment:
Personal taste, but I would prefer these flags to be moved to the same place as all other flags. It would make it easier for a reader to see what flags are possible.